extensions/wikia/FlowTracking/scripts/createNewPageFlows.js

```diff
@@ -0,0 +1,27 @@
+define('wikia.flowTracking.createPage',
+	['wikia.flowTracking', 'wikia.querystring', 'mw', 'wikia.document'],
+	function (flowTrack, QueryString, mw, d)
+{
+	var namespaceId = mw.config.get('wgNamespaceNumber'),
+		articleId = mw.config.get('wgArticleId');
+
+	function trackOnEditPageLoad(editor) {
+		// Track only creating articles (wgArticleId=0) from namespace 0 (Main)
+		// IMPORTANT: on Special:CreatePage even after providing article title the namespace is set to -1 (Special Page)
+		if (namespaceId === 0 && articleId === 0) {
+			var qs = new QueryString(),
+				// 'flow' is the parameter passed in the url if user has started a flow already
+				flowParam = qs.getVal('flow', false);
+
+			if (flowParam || d.referrer) {
+				//TODO: track middle step for other flows
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1135)